env_support/cmsis-pack/lv_conf_cmsis.h

```diff
@@ -578,6 +585,14 @@
 /*Enable the examples to be built with the library*/
 #define LV_BUILD_EXAMPLES 1
 
+/*==================
+* DEMOS
+*==================*/
+
+#if LV_USE_DEMO_WIDGETS && LV_MEM_CUSTOM == 0 && LV_MEM_SIZE < (35ul * 1024ul)
```

Comment:
We can move this into `lv_demo_widgets.c` too.

Reply:
Sure, this is the best option. Will do that.

Reply:
For all cmsis-pack related PR, I tried to avoid changing any content outside the cmsis-pack folder.

So for this change, can you please merge this PR first. I will do it in a separate PR.

Reply:
Okay, merging.